py/gc.c

```diff
@@ -1059,7 +1059,7 @@ void gc_dump_alloc_table(void) {
                 // print header for new line of blocks
                 // (the cast to uint32_t is for 16-bit ports)
                 // mp_printf(&mp_plat_print, "\n%05x: ", (uint)(PTR_FROM_BLOCK(area, bl) & (uint32_t)0xfffff));
-                mp_printf(&mp_plat_print, "\n%05x: ", (uint)((bl * BYTES_PER_BLOCK) & (uint32_t)0xfffff));
+                mp_printf(&mp_plat_print, "\n%07x: ", (uint)((bl * BYTES_PER_BLOCK) & (uint32_t)0xfffffff));
```

Comment:
The mask can be removed, it shouldn't be there.

Then I think we should just make this format `%08x` so it supports >256MiB.

Reply:
Updated.